charts/README.md

```diff
@@ -49,9 +49,19 @@ To disable Orbit, use:
 If a custom Orbit endpoint is required, use:
 
 ```shell
---set cloudserver-front.orbit.endpoint=https://dev.zenko.io
+--set cloudserver-front.orbit.endpoint=https://custom.orbit.endpoint
+--set cloudserver-front.orbit.pushEndpoint=https://custom.push.orbit.endpoint
 ```
 
+and if `poll` mode is needed, use:
+```shell
+--set cloudserver-front.orbit.mode=poll
+```
+
+**Note:** In `poll` mode, Cloudserver will request changes from Orbit instead
+of receiving updates when one become available, so the `orbit.pushEndpoint`
+value will be ignored.
```

Comment:
I wasn't going to expose this setting in the docs but since we are, a couple of things are worth mentioning as well. In poll mode:
- interactive features such as Multicloud Browser, Metadata Search, CRR object status and CRR object retry are not available
- configuration updates may take up to a minute to propagate
- statistics reports are sent less frequently

Poll mode should be used when long-lived websocket connections are not available.